src/hosting/implicitInit.ts

```diff
@@ -8,8 +8,6 @@ import { EmulatorRegistry } from "../emulator/registry";
 import { EMULATORS_SUPPORTED_BY_USE_EMULATOR, Emulators } from "../emulator/types";
 import { readTemplateSync } from "../templates";
 
-const INIT_TEMPLATE = readTemplateSync("hosting/init.js");
```

Comment:
Had to move this into the function so that it wasn't evaluated before activation.

Reply:
It's exactly this kind of stuff that makes me hate mutable state so much that I even awakened the mighty webpack. Well, at least we know what the tradeoffs are and we can deal with it another day